code/modules/mob/living/simple_animal/friendly/pet.dm

```diff
@@ -4,21 +4,23 @@
 	var/obj/item/clothing/neck/petcollar/pcollar = null
 	var/collar = ""
 	var/pettag = ""
+	var/canwear = 1
```

Comment:
this var should be named `can_wear_pet_collar` or something similar, canwear is too ambiguous and vague